### ISSUE TITLE
Question about hidden layers from pretained model

### ISSUE THREAD

**Issue by mvss80** [huggingface/pytorch-pretrained-BERT#167]
In the example shown to get hidden states https://github.com/huggingface/pytorch-pretrained-BERT#usage
I want to confirm - the final hidden layer corresponds to the last element of `encoded_layers`, right?

**Comment by rodgzilla** [comment]
Yes you are right. The first value returned is the output for `BertEncoder.forward`.

https://github.com/huggingface/pytorch-pretrained-BERT/blob/8da280ebbeca5ebd7561fd05af78c65df9161f92/pytorch_pretrained_bert/modeling.py#L623-L634
